src/App.tsx

```diff
@@ -21,6 +41,8 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={itemsPerPage}
+            onChange={(event) => selectPerPage(event)}
```

Comment:
```suggestion
            onChange={ selectPerPage}
```
enough to pass reference on handler